qemu-3dfx Darwin build fix: GL_CONTEXTALPHA was defined only under
CONFIG_LINUX but is used in the shared Linux/Darwin body of
MGLPresetPixelFormat() (upstream regression, qemu-3dfx d00e858).
Error on macOS: "use of undeclared identifier GL_CONTEXTALPHA".

--- a/hw/mesa/mglcntx_linux.c	2026-09-02 09:03:34.727139621 -0300
+++ b/hw/mesa/mglcntx_linux.c	2026-09-02 09:03:34.742430759 -0300
@@ -71,8 +71,9 @@
 
     return ret;
 }
-#define GL_CONTEXTALPHA 1
 #endif /* CONFIG_LINUX */
+/* used by MGLPresetPixelFormat() below on both Linux and Darwin */
+#define GL_CONTEXTALPHA 1
 
 typedef uint16_t WORD;
 typedef uint32_t DWORD;
